content/navigationBar/navMenu.json

```
@@ -2,47 +2,84 @@
   "navigationBar": "Master",
   "navItem": [
     {
-      "label": "Products",
+      "label": "TinaCMS",
       "items": [
         {
-          "labelLeft": "TinaCMS",
-          "hrefLeft": "/tinacms",
-          "labelRight": "Docs",
-          "hrefRight": "/docs"
+          "label": "Home",
+          "href": "/tinacms"
         },
         {
-          "labelLeft": "TinaDocs",
-          "hrefLeft": "/tinadocs",
-          "labelRight": "Docs",
-          "hrefRight": "/tinadocs/docs"
+          "label": "Docs for TinaCMS",
+          "href": "/docs"
+        },
+        {
+          "label": "Videos",
+          "href": "https://www.youtube.com/c/tinacms"
+        },
+        {
+          "label": "Compare TinaCMS",
+          "href": "/compare-tina"
+        },
+        {
+          "label": "Showcase",
+          "href": "/showcase"
+        },
+        {
+          "label": "What's New in TinaCMS",
+          "href": "/whats-new/tinacms"
         }
       ],
-      "_template": "doubleNavItemDropDown"
+      "_template": "groupOfStringItems"
     },
     {
-      "label": "Cloud Pricing",
-      "href": "/pricing",
-      "_template": "stringItem"
+      "label": "TinaDocs",
+      "items": [
+        {
+          "label": "Home",
+          "href": "/tinadocs"
+        },
+        {
+          "label": "Docs for TinaDocs",
+          "href": "/tinadocs/docs"
+        }
+      ],
+      "_template": "groupOfStringItems"
     },
     {
-      "label": "Community",
+      "label": "TinaCloud",
       "items": [
         {
-          "label": "Showcase",
-          "href": "/showcase"
+          "label": "My TinaCloud",
+          "href": "https://app.tina.io/signin"
         },
         {
-          "label": "Roadmap",
-          "href": "/roadmap"
+          "label": "What's New in TinaCloud",
+          "href": "/whats-new/tinacloud"
+        }
+      ],
+      "_template": "groupOfStringItems"
+    },
+    {
+      "label": "Pricing",
+      "items": [
+        {
+          "label": "Tina Cloud Pricing",
```

TinaCloud is one word 
```suggestion
          "label": "TinaCloud Pricing",
```